imports/api/crud-controller/module/methods.js

```diff
@@ -46,6 +59,12 @@ export const insertOneModuleInSemester = function insertOneModuleInSemester(seme
   if (!retrievedSemesters) {
     return {};
   }
+
+  // checks semester index is not larger than the size of semester
```

Comment:
yea just a check to see if the input index is actually larger than the number of semesters in currently in the planner